omod/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestControllerTest.java

```diff
@@ -124,20 +107,6 @@ public void shouldGetRefByUuid() {
 	public void shouldGetFullByUuid() {
 	}
 	
-	@Test
-	public void saveAttachment_shouldUpdateObsComment() throws Exception {
-		// Setup
-		String editedComment = "Hello world!";
-		String json = "{\"uuid\":\"" + getUuid() + "\",\"comment\":\"" + editedComment + "\"}";
```

Comment:
Don't you need this test?

Reply:
This test fails currently for all 2.x versions of openmrs platform because of this [interceptor here](https://github.com/openmrs/openmrs-core/blob/b7c03a28c64493023a8211c53bb60d4d944b39b7/api/src/main/java/org/openmrs/api/db/hibernate/ImmutableObsInterceptor.java#L26).